api/room_resource/schema.py

```diff
@@ -226,6 +231,15 @@ def resolve_get_resources_by_room_id(self, info, room_id):
             resources.append(room_resource)
         return RoomResources(roomResources=resources)
 
+    @Auth.user_roles('Admin')
+    def resolve_rooms_containing_resource(self, info, resource_id):
+        query = RoomResource.get_query(info).join(ResourceModel).join(
+          RoomModel)
+        rooms_with_resource = query.filter(
+          RoomResourceModel.resource_id == resource_id).filter(
+          ResourceModel.state == "active").filter(RoomModel.state == "active")
```

Comment:
Nice work @rajeman However, why did you choose to call filter twice on the same model while you could have called it once like so 
```
rooms_with_resource = query.filter(
                RoomResourceModel.resource_id == resource_id,
                ResourceModel.state == "active").filter(RoomModel.state == "active")
```

Reply:
Thanks. I will refactor as suggested.